src/booking/models/booking-date.enum.ts

```diff
@@ -11,3 +11,32 @@ export enum BookingTimePageText {
   CAPTION = 'Work access',
 }
 
+export enum BookingDateFormErrorMessage {
+  BOOKING_START_DATE_FAILED = 'Enter a valid booking start date',
+  BOOKING_END_DATE_FAILED = 'Enter a valid booking end date',
+  BOOKING_START_DATE_EMPTY_CHECK_FAILED = 'Enter a booking start date ',
+  BOOKING_END_DATE_EMPTY_CHECK_FAILED = 'Enter a booking end date',
+  BOOKING_DATE_COMPARISON_FAILED = 'The booking end date cannot be earlier than the booking start date',
+  BOOKING_DATE_INPUT_CHECK_FAILED = 'Please Enter Valid Date',
+  PAST_DATE_CHECK_FAILED = 'The booking start date cannot be in the past',
+  BOOKING_START_DATE_EMPTY_CHECK = 'booking-start-date-empty-check',
+  BOOKING_END_DATE_EMPTY_CHECK = 'booking-end-date-empty-check',
+  BOOKING_BOTH_DATE_EMPTY_CHECK = 'booking-Both-date-empty-check',
```

Comment:
Do these need to be error messages instead of hyphenated? Also why is Both capitalised?